### PR TITLE
MAT-7792, MAT-7793: Set up Functions - Expresion Edtior section

### PR DESCRIPTION
**Pull request by mcmcphillips**
## MADiE PR

Jira Ticket: [MAT-7792](https://jira.cms.gov/browse/MAT-7792)
Jira Ticket: [MAT-7793](https://jira.cms.gov/browse/MAT-7793)

(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is in to the **correct branch**.
* [x] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [x] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

**Review by adongare — DISMISSED** [review]

**Review by chubert-sb — DISMISSED** [review]

File: src/CqlBuilderPanel/common/utils.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+// import { formatExpressionName } from "../definitionsSection/definitionBuilder/DefinitionBuilder";
```
Comment (RohitKandimalla) [inline src/CqlBuilderPanel/common/utils.tsx]: I think we can remove this

File: src/CqlBuilderPanel/common/utils.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  return values?.type !== "Timing" && values?.type !== "Pre-Defined Functions"
+    ? values?.type === "Functions" || values?.type === "Fluent Functions"
+      ? values?.name?.replace(/([\w\s]+)\(\)/g, '"$1"()')
+      : values?.name.includes(".")
+      ? values?.name.replace(/(.*\.)(.*)/, '$1"$2"')
+      : `"${values?.name}"`
+    : values?.name;
+};
+// given
+export const getNewExpressionsAndLines = (
```
Comment (RohitKandimalla) [inline src/CqlBuilderPanel/common/utils.tsx:11]: Do you think we will have value adding a unit test suite for this Util class?

File: src/CqlBuilderPanel/functionsSection/functionBuilder/FunctionBuilder.tsx [hunk 3]
```diff
@@ -23,11 +26,14 @@ export interface Funct {
...
 
 export interface FunctionProps {
+  cqlBuilderLookupsTypes: CqlBuilderLookup;
   canEdit: boolean;
   handleApplyFunction: Function;
   handleFunctionEdit?: Function;
   funct?: Funct;
   onClose?: Function;
+  operation?: string;
+  cqlFunction?: any;
```
Comment (RohitKandimalla) [inline src/CqlBuilderPanel/functionsSection/functionBuilder/FunctionBuilder.tsx]: Are these required for the future stories ?

**Review by RohitKandimalla — DISMISSED** [review]
Minor observations, not necessarily have to be modified.

File: src/CqlBuilderPanel/common/utils.tsx (hunk 2 above)
Comment (mcmcphillips) [reply src/CqlBuilderPanel/common/utils.tsx:11]: Possibly if a lot more is added to it. Right now It's just covered in it's own describe in another suite taht uses it

File: src/CqlBuilderPanel/functionsSection/functionBuilder/FunctionBuilder.tsx (hunk 3 above)
Comment (mcmcphillips) [reply src/CqlBuilderPanel/functionsSection/functionBuilder/FunctionBuilder.tsx]: operation should be for edit capability. I think cqlFunction can be removed.

**Review by sb-cecilialiu — APPROVED** [review]
Looks good to me.

**Pull request merged**